hive-data-gen/src/main/java/com/wipro/ats/bdre/datagen/GeneratorMain.java

```diff
@@ -36,7 +36,7 @@ public static void main(String[] args) throws Exception {
         java.util.Properties listForParams= getProperties.getProperties(processId,"table");
          String outputDir=listForParams.getProperty("outputPath");
 
-        String params[] =new String[]{processId,outputDir};
+        String[] params =new String[]{processId,outputDir};
         int res = ToolRunner.run(new Configuration(), new Driver(), params);
         System.exit(res);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this call to "exit" or ensure it is really required. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1147)